crates/sui-framework/sources/bcs.move

```diff
@@ -1,7 +1,20 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+/// This module implements BCS (de)serialization in Move.
+/// Full specification can be found here: https://github.com/diem/bcs
+///
+/// Short summary (for Move-supported types):
+///
+/// - address - sequence of X bytes
+/// - bool - byte with 0 or 1
+/// - u8 - a single u8 byte
+/// - u64 / u128 - LE bytes
+/// - vector - ULEB length + LEN elements
```

Comment:
nit: I know everyone probably can infer what "ULEB" means here, but it technically doesn't make sense without the "128", because it stands for "Unsigned Little-Endian, Base 128"